airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -120,16 +122,46 @@ def get_fields_from_schema(schema: Mapping[str, Any]) -> List[str]:
 
     @staticmethod
     def convert_schema_into_query(
-        schema: Mapping[str, Any], report_name: str, from_date: str = None, to_date: str = None, cursor_field: str = None
+        schema: Mapping[str, Any],
+        report_name: str,
+        from_date: str = None,
+        to_date: str = None,
+        cursor_field: str = None,
+        id_list: List[str] = None,
+        id_field: str = None,
+        resource_type: str = None,
+        limit: int = None,
     ) -> str:
         from_category = REPORT_MAPPING[report_name]
         fields = GoogleAds.get_fields_from_schema(schema)
+        if report_name != "change_status" and "change_status.last_change_date_time" in fields:
+            fields.remove("change_status.last_change_date_time")
         fields = ", ".join(fields)
 
         query_template = f"SELECT {fields} FROM {from_category}"
 
+        conditions = []
+
+        # time condition
+        if cursor_field and from_date and to_date:
+            conditions.append(f"{cursor_field} >= '{from_date}' AND {cursor_field} <= '{to_date}'")
+
+        # filter by ids if needed
+        if id_list and id_field:
+            id_list_str = ", ".join(f"'{str(id_)}'" for id_ in id_list)
+            conditions.append(f"{id_field} IN ({id_list_str})")
+
+        if resource_type:
```

Comment:
please add comment like in line 149 with condition description

Reply:
Added